pkg/network/ebpf/c/protocols/kafka/kafka-classification.h

```diff
@@ -165,9 +164,37 @@ static __always_inline bool skip_varint_number_of_topics(pktbuf_t pkt, u32 *offs
     return true;
 }
 
+static __always_inline __maybe_unused bool skip_varint(pktbuf_t pkt, u32 *offset, u32 max_bytes) {
+    u8 bytes[4] = {};
+
+    if (*offset + max_bytes > pktbuf_data_end(pkt)) {
+        return false;
+    }
+
+    pktbuf_load_bytes(pkt, *offset, bytes, max_bytes);
+
+    u32 i = 1;
+    u8 current_byte = bytes[0];
+
+    #pragma unroll
+    for (; i < max_bytes; i++) {
+        *offset += 1;
+        if (!isMSBSet(current_byte)) {
+            break;
+        }
+        current_byte = bytes[i];
+    }
```

Comment:
Does pragma unroll work good with "a variable" length?
is it a "replacement" for macros?